src/program/OutfitStudio.cpp

```diff
@@ -869,7 +869,11 @@ OutfitStudioFrame::OutfitStudioFrame(const wxPoint& pos, const wxSize& size) {
 		lightDirectional2Slider->SetValue(directional2);
 	}
 
+	wxPanel* m_panel6 = (wxPanel*)FindWindowByName("m_panel6");
+	m_panel6->SetBackgroundColour(wxColour(112, 112, 112));
```

Comment:
What's m_panel16 and why is this required? Is there a way to do it more cleanly?

Reply:
I tried setting m_panel6's background color in the xrc, and wxWidgets for gtk ignored it.  Without the background color being set, all of the text labels in the panel are white on an off-white background for me.  (That's "De-/Select Sliders", "Fixed Weight Brush", "X-Mirror Bone", and "Preview Scaling".)

Reply:
The parent panel of m_panel6 is bottomSplitPanel.  I tried setting bottomSplitPanel's background color instead to see if the color propagates to the child panel, and it doesn't.  The panel that needs its color set is m_panel6.